app/Http/Requests/FilterPublicCampaignRequest.php

```diff
@@ -29,7 +29,8 @@ public function rules()
             'featured_until' => 'nullable|boolean',
             'is_open' => 'nullable|boolean',
             'system' => 'nullable|string|max:25',
-            'language' => 'nullable|string|max:5'
+            'language' => 'nullable|string|max:5',
+            'genre' => 'nullable|string|max:25'
```

Comment:
This says it's a string max 25 but in the code it looks like it's testing it on the id in `scopeFilterPublic`